src/whathappened/sheets/schema/dod.py

```diff
@@ -0,0 +1,213 @@
+
+    utmattad: bool = False
+    krasslig: bool = False
+    omtocknad: bool = False
+    arg: bool = False
+    radd: bool = False
+    uppgiven: bool = False
+
+
+class Skadebonus(msgspec.Struct, frozen=True):
+    sty: Bonus = "-"
+    smi: Bonus = "-"
+
+
+class Fardighet(msgspec.Struct, frozen=True):
+    name: str
+    base: str
+    checked: bool = False
+    value: Attribut = 0
+
+
+PrimaraFerdigheter = [
+    ("Bestiologi", "INT"),
+    ("Bluffa", "KAR"),
+    ("Fingerfärdighet", "SMI"),
+    ("Finna dolda ting", "INT"),
+    ("Främmande språk", "INT"),
+    ("Hantverk", "STY"),
+    ("Hoppa & klättra", "SMI"),
+    ("Jakt & fiske", "SMI"),
+    ("Köpslå", "KAR"),
+    ("Läkekonst", "INT"),
+    ("Myter & legender", "INT"),
+    ("Rida", "SMI"),
+    ("Simma", "SMI"),
+    ("Sjökunnighet", "INT"),
+    ("Smyga", "SMI"),
+    ("Undivka", "SMI"),
+    ("Uppträda", "KAR"),
+    ("Upptäcka fara", "INT"),
+    ("Vildmakrsvana", "KAR"),
+]
+
+VapenFardigheter = [
+    ("Armbrost", "SMI"),
+    ("Hammare", "STY"),
+    ("Kniv", "SMI"),
+    ("Pilbåge", "SMI"),
+    ("Slagsmål", "STY"),
+    ("Slunga", "SMI"),
+    ("Spjut", "STY"),
+    ("Stav", "SMI"),
+    ("Svärd", "STY"),
+    ("Yxa", "STY"),
+]
+
+
+class Fardigheter(msgspec.Struct, frozen=True):
+    primar: List[Fardighet] = msgspec.field(
+        default_factory=lambda: [
+            Fardighet(name=name, base=base) for name, base in PrimaraFerdigheter
+        ]
+    )
+    vapenfardigheter: List[Fardighet] = msgspec.field(
+        default_factory=lambda: [
+            Fardighet(name=name, base=base) for name, base in VapenFardigheter
+        ],
+        name="Vapenfärdigheter",
+    )
+
+    sekundarafardigheter: List[Fardighet] = msgspec.field(
+        default_factory=lambda: [], name="Sekundära färdigheter"
+    )
+
+
+class Packning(msgspec.Struct, frozen=True):
+    barformoga: int = msgspec.field(default=0, name="Bärformåga")
+    items: List[str] = []
+    minnessak: str = ""
+    smaasaker: List[str] = []
+
+
+class Penger(msgspec.Struct, frozen=True):
+    guldmynt: int = 0
+    silvermynt: int = 0
+    kopparmynt: int = 0
+
+
+class Rustning(msgspec.Struct, frozen=True):
+    skyddsvärde: int = 0
+    smyga: bool = False
+    undvika: bool = False
+    hoppa_och_klatra: bool = False
+
+
+class Hjalm(msgspec.Struct, frozen=True):
+    skyddsvärde: int = 0
+    upptäcka_fara: bool = False
+    avståndsattacker: bool = False
+
+
+class Vapen(msgspec.Struct, frozen=True):
+    vapen: str = ""
+    grepp: str = ""
+    räckvidd: str = ""
+    skada: str = ""
+    brytvärde: str = ""
+    egenskaper: str = ""
+
+
+class Bevapning(msgspec.Struct, frozen=True):
+    rustning: Rustning = msgspec.field(default_factory=Rustning)
+    hjalm: Hjalm = msgspec.field(default_factory=Hjalm, name="Hjälm")
+    til_hands: List[Vapen] = msgspec.field(
+        default_factory=lambda: [Vapen() for _ in range(3)]
+    )
+
+
+class Viljepoang(msgspec.Struct, frozen=True):
+    poeng: int = 0
```

Comment:
In case you want to stay consistent with Swedish for the properties, these are called "poäng" and "använda", respectively.